Sources/Basic/OrderedDictionary.swift

```diff
@@ -78,6 +78,17 @@ public struct OrderedDictionary<Key: Hashable, Value> {
         array.remove(at: array.index(of: key)!)
         return value
     }
+
+    /// An array containing just the values of the ordered dictionary.
+    public var values: [Value] {
+        return self.array.compactMap { self.dict[$0] }
```

Comment:
I think this should be a map since it is an error not to have an element for a key.

Reply:
With a force-unwrap? Both options are just to please the compiler, since OrderedDict shouldn't make it possible to have one but not the other, but a force-unwrap wouldn't hide the error, that's true 👍